pkg/v5/key/key.go

```diff
@@ -11,6 +11,11 @@ import (
 	apismetav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+// return alt-names for api certs
```

Comment:
Godoc is funny here. Should be

`APIAltNames returns the alt names for API certs.`

Reply:
Yes also APIAltNames should take in clusterGuestConfig as a param. You can call ClusterID in APIAltNames.

Reply:
@rossf7 i followed similar key function that is also used in this scope  `CertConfigName` which also takes only string

Reply:
OK I'd say that one was wrong too but its not a blocker.